iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/utils/Utils.java

```diff
@@ -151,7 +152,7 @@ public static ConsensusGroupId fromRaftGroupIdToConsensusGroupId(RaftGroupId raf
     return ConsensusGroupId.Factory.create((int) type, byteBuffer.getInt());
   }
 
-  public static ByteBuffer serializeTSStatus(TSStatus status) throws TException {
+  public static ByteBuffer serializeStatus(TSStatus status) throws TException {
```

Comment:
TSStatus is one of our idioms, so we don't have to change the name

Reply:
All fixed